Mathlib/Analysis/Analytic/Linear.lean

```diff
@@ -77,30 +77,26 @@ def fpowerSeriesBilinear (f : E →L[𝕜] F →L[𝕜] G) (x : E × F) : Formal
   | 2 => f.uncurryBilinear
   | _ => 0
 
+@[simp]
```

Comment:
Hmmm, I'm not a big fan of the RHS's of this lemma and the next.
Are we forced to have this in the default simp-set?

Reply:
Aah, I only now realise that before this the def itself had a simp attribute.